frontend/src/components/Jobs.vue

```diff
@@ -106,26 +138,40 @@ export default {
     fetchData() {
       return fetchCctrayJson()
           .then(this.marshalData)
-          .then((data) => this.jobs = data)
+          .then((data) => {
+            this.visibleJobs = data.filter(j => !preferences.isJobHidden(j.name))
+            this.hiddenJobs = data.filter(j => preferences.isJobHidden(j.name))
+          })
           .catch((reason) => {
             console.error(reason);
             return Promise.reject(reason);
           });
+    },
+    toggleHiddenJobs() {
+      this.showHiddenJobs = !this.showHiddenJobs
     }
   }
 }
 </script>
 
 <style scoped>
 
-#jobs {
+.jobs {
   font-family: "OpenSans", sans-serif;
   display: grid;
   align-items: center;
   grid-template-rows: repeat(auto-fit, minmax(90px, 115px));
   grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
   grid-gap: 10px;
   width: 100%;
+  margin-bottom: 3em;
```

Comment:
Instead of having **margin-bottom** for jobs, we should use **margin-top** for hidden jobs

Reply:
Changed accordingly